src/containers/StudyEdit/StudyEditForm.tsx

```diff
@@ -64,70 +99,117 @@ function StudyEditForm() {
+
+        formData.append('title', values.title);
+        if (imageSrc) formData.append('imageFile', imageSrc);
+        formData.append('description', values.description);
+
+        setTimeout(() => {
+          for (const key of formData.keys()) {
+            console.log(key, ':', formData.get(key));
+          }
+          actions.setSubmitting(false);
+        }, 3000);
+      }}
+    >
+      {(formik) => {
+        return (
+          <form onSubmit={formik.handleSubmit}>
+            <StudyEditWrapper>
+              <StudyEditHeading>
+                <Typography variant='h4'>스터디 수정</Typography>
+                <InputWrapper>
+                  {formik.touched.title && formik.errors.title ? (
+                    <ErrorMessage ref={titleRef}>
+                      {(() => {
+                        if (titleRef.current) {
+                          titleRef.current.scrollIntoView({
+                            behavior: 'smooth',
+                            block: 'start',
+                          });
+                        }
+                        return formik.errors.title;
+                      })()}
+                    </ErrorMessage>
+                  ) : null}
+                  <Field
+                    as={LabelInput}
+                    label='스터디명'
+                    id='title'
+                    {...formik.getFieldProps('title')}
+                  />
+                </InputWrapper>
+              </StudyEditHeading>
+              <StudyEditImageWrapper>
+                <Typography variant='h5'>대표 이미지</Typography>
+                <ImageContainer>
+                  {fileErrorMessage && (
+                    <ErrorMessage>{fileErrorMessage}</ErrorMessage>
+                  )}
+                  <ImageWrapper>
+                    {thumbnailImage && !fileErrorMessage ? (
+                      <Image src={thumbnailImage} alt='study-image' />
+                    ) : (
+                      <CameraIcon color='secondary' />
+                    )}
+                  </ImageWrapper>
+                </ImageContainer>
+                <ButtonWrapper>
+                  <FileInput message='이미지 업로드' onChange={onImageChange} />
+                </ButtonWrapper>
+              </StudyEditImageWrapper>
+              <StudyDescriptionWrapper>
+                <Typography variant='h5'>상세 설명</Typography>
+                {formik.touched.description && formik.errors.description ? (
+                  <ErrorMessage ref={descriptionRef}>
+                    {(() => {
+                      if (descriptionRef.current && !titleRef.current) {
+                        descriptionRef.current.scrollIntoView({
+                          behavior: 'smooth',
+                          block: 'start',
+                        });
+                      }
+                      return formik.errors.description;
+                    })()}
+                  </ErrorMessage>
+                ) : null}
```

Comment:
p4;
&&을 이용해서 null을 없애도 좋을 것 같습니다. :) 

```suggestion
                {formik.touched.description && formik.errors.description && (
                  <ErrorMessage ref={descriptionRef}>
                    {(() => {
                      if (descriptionRef.current && !titleRef.current) {
                        descriptionRef.current.scrollIntoView({
                          behavior: 'smooth',
                          block: 'start',
                        });
                      }
                      return formik.errors.description;
                    })()}
                  </ErrorMessage>
                ) }
```

Reply:
jsx 조건문 수정 완료했습니다. 감사합니다. 👍 847212d54520b9315230c1c934b7117bc671683b